### ISSUE TITLE
Remove setter/getter <method> tags for properties in doc/classes/*.xml

### ISSUE THREAD

**Issue by NathanLovato** [godotengine/godot#12278]
Setters and getters linked to <member> tags are ignored when you build the docs, but the <method> tags are still in the docs' source .xml file.

We kept them to move some existing descriptions to <member> tags. Now even if we tell contributors not to document them, some still do it or never get in touch with active doc writers and don't read the writing guidelines.

Now Godot 3's class ref completion is higher than Godot 2, could we remove them from the doctool generation?

Example from the Control class:
```xml
<member name="hint_tooltip" type="String" setter="set_tooltip" getter="_get_tooltip">
  Changes the tooltip text. The tooltip appears when the user's mouse cursor stays idle over this control for a few moments.
</member>
```

This is in Control.xml but not in the final docs:
```xml
<method name="set_tooltip">
	<return type="void">
	</return>
	<argument index="0" name="tooltip" type="String">
	</argument>
	<description>
		Changes the tooltip text. The tooltip appears when the user's mouse cursor stays idle over this control for a few moments.
	</description>
</method>
[...]
<method name="get_tooltip" qualifiers="const">
	<return type="String">
	</return>
	<argument index="0" name="at_position" type="Vector2" default="Vector2( 0, 0 )">
	</argument>
	<description>
		Return the tooltip, which will appear when the cursor is resting over this control.
	</description>
</method>
```

**Comment by NathanLovato** [comment]
Even if we're around the entire week, it's almost 1 PR out of 2 or 3 where contributors write methods that won't be in the docs. Example from today: https://github.com/godotengine/godot/pull/12339/files
And it's every week, almost every new contributor, even if it's in the writing guidelines and we told people quite a few times.

**Comment by StraToN** [comment]
Since getter and setter functions can be automatically determined (`set_<variable>()`) I'd fully agree.

**Comment by LikeLakers2** [comment]
If we're gonna do this, then we'll also want to make sure that people are giving the variable descriptions some decent descriptions. Things like [Node2D's global_scale variable](https://github.com/godotengine/godot/blob/master/doc/classes/Node2D.xml#L315-L317)'s description are way too short. (What is the global scale? How is it different from regular scaling?)

**Comment by cbscribe** [comment]
Agreed, but it's not like the setter is any better: "Sets the node's global scale." We're not *losing* any information. And at least this way, there's one description that needs to be improved.

**Comment by NathanLovato** [comment]
@LikeLakers2 Sure but that's another problem. The set/get descriptions I'm talking about are in the source file, but they're not part of the online or in engine docs.
For member variables, we have some writing guidelines now. Contributors focus on filling empty docs and with 10 000 descriptions to write, we haven't had time to review most of the existing ones yet. We'd need more writers to help.

**Comment by LikeLakers2** [comment]
@NathanLovato I'm just saying that once we *do* get to removing the get/set methods, we should make sure we aren't also removing a decent description that we could be placing in the member variable's description.

**Comment by reduz** [comment]
you dont need to do any hack to determine them, Godot API that generates
the xml can easily do it. I can try working on this when I get back home

**Comment by NathanLovato** [comment]
@LikeLakers2 There were almost none in the 3000 entries we filled in the class reference so far + we filled the majority of the corresponding members by now (Godot 2.x's classref is ~60% complete, Godot 3.0 66-67% complete already)

**Comment by akien-mga** [comment]
Fixed with f6139c2c, 31f62f5 and 92f07fefc. 🎉
